projects/media-viewer/src/lib/media-viewer/viewers/pdf-viewer/pdf-viewer.component.ts

```diff
@@ -42,7 +42,9 @@ export class PdfViewerComponent implements AfterViewInit {
   async ngAfterViewInit(): Promise<void> {
     this.pdfWrapper = this.pdfJsWrapperFactory.create(this.viewerContainer);
     this.pdfWrapper.currentPageChanged.subscribe(v => this.currentPageChanged.next(v));
-    this.pdfWrapper.searchResults.subscribe(v => this.searchResults.next(v));
+    this.pdfWrapper.searchResults.subscribe(v => {
```

Comment:
I think you should probably undo this one, it's fine as is

Reply:
done, I did it cause I stuck a console log in there earlier